File: level2/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''

    s = s.split(' ')
    print(s)
    for i in s:
        if i == '':
            answer = answer + ' '
            continue
        if i[0].isdigit():
            a = i[0]
        else:
            a = i[0].upper()
        b = i[1:].lower()
        answer = answer + a + b + " "

    return answer[:-1]

print(solution("3people unFollowed me"))
print(solution("for the last week"))
print(solution('a b c d ed f easg sdgacvsd dsgawe'))
print(solution(' 3 2 1 zero i am  k ing '))

File: level2/JadenCase 문자열 만들기.js
function solution(s) {
  const targets = s.split('');
  let answer = '';
  let before = ' ';

  targets.forEach(target => {
    if (target === ' ') {
      answer += target;
      before = ' ';
    }
    else if (before === ' ') {
      answer += target.toUpperCase();
      before = target;
      return;
    }
    else {
      answer += target.toLowerCase();
      before = target;
      return;
    }
  })

  return answer;
}